src/tools/lib/python/picongpu/plugins/energy_histogram.py

```diff
@@ -0,0 +1,160 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017 PIConGPU contributors
+Authors: Axel Huebl
+License: GPLv3+
+"""
+
+import numpy as np
+import os
+
+
+class EnergyHistogram(object):
+    """
+    Data Reader for the Energy Histogram Plugin.
+    """
+
+    def __init__(self, run_directory):
+        """
+        Parameters
+        ----------
+        simulation_directory : string
+            path to the run directory of PIConGPU
+            (the path before ``simOutput/``)
+        """
+        if run_directory is None:
+            raise ValueError('The run_directory parameter can not be None!')
```

Comment:
The check is done on first data access.
Due to the species' influence, it also makes no sense to check it earlier (dir + file)